db/controllers/articles-controller.js

```diff
@@ -35,12 +36,15 @@ exports.postArticleComment = (req, res, next) => {
         next(err)
     })
 }
-// exports.updateArticleById = (req, res, next) => {
-//     updateArticleById()
-//     .then(() => {
-
-//     })
-//     .catch((err) => {
-//         next(err)
-//     })
-// }
\ No newline at end of file
+exports.patchArticleById = (req, res, next) => {
+    const { inc_votes } = req.body
+    const { article_id } = req.params;   
+    const allPromises = Promise.all([ fetchArticlesById(article_id), updateArticleById(inc_votes, article_id)])
+    allPromises.then((result) => {       
+        const article = result[1].rows[0]         
```

Comment:
Ideally, this should happen within the model, not the controller, so that whatever we're returning from the model is as simple as possible to send in the controller :)

Reply:
done